epsagon/trace.py

```diff
@@ -211,11 +211,12 @@ def _create_new_trace(self, unique_id=None):
             url_patterns_to_ignore=self.url_patterns_to_ignore,
             keys_to_ignore=self.keys_to_ignore,
             keys_to_allow=self.keys_to_allow,
-            unique_id=unique_id,
+            transport=self.transport,
```

Comment:
This is a bug fix - currently, if the user changes the default transport it won't be used in many cases